packages/core/src/routes/hook.ts

```diff
@@ -21,23 +24,59 @@ export default function hookRoutes<T extends AuthedRouter>(
     }
   );
 
+  router.get(
+    '/hooks/:id',
+    koaGuard({
+      params: z.object({ id: z.string() }),
+      response: Hooks.guard,
+      status: [200, 404],
+    }),
+    async (ctx, next) => {
+      const {
+        params: { id },
+      } = ctx.guard;
+
+      ctx.body = await findHookById(id);
+
+      return next();
+    }
+  );
+
   router.post(
     '/hooks',
-    koaGuard({ body: Hooks.createGuard.omit({ id: true }), response: Hooks.guard, status: 200 }),
+    koaGuard({
+      body: Hooks.createGuard
+        .omit({ id: true, name: true, signingKey: true, enabled: true, events: true })
```

Comment:
maybe there's no need to explicitly omit `enabled` on the API layer?

Reply:
just give it a default value in guard

Reply:
Make sense. Allow users to create a disabled webhook by this API.